Sources/Kodable/KodableTransformable.swift

```diff
@@ -22,12 +22,12 @@ public protocol KodableTransform {
     public typealias OriginalType = T.From
     public typealias TargetType = T.To
 
-    // Note: this might crash if an instance of a type uses the property wrapper with a non-optional type
-    /// and it is not decoded. (i.e. the `_value` is not initialized a this it is nil.)
     private func _wrappedValue<U>(_: U.Type) -> U {
         _value as! U
```

Comment:
```suggestion
        if _value == nil {
            fatalError(message: "Property with key '\(pending-property-key-here)' couldn't be decoded, and no default value was provided.")
        } else {
            _value as! U
        }
```

Just missing the property key as idk how to get it within this context @JARMourato 🙈